tests/e2e_args.py

```diff
@@ -100,6 +100,9 @@ def cli_args(add=lambda x: None, parser=None, accept_unknown=False):
     parser.add_argument(
         "--label", help="Unique identifier for the test", default=default_label
     )
+    parser.add_argument(
+        "--pbft", help="Use the PBFT consensus", action="store_true",
```

Comment:
I think the description should be something along "Opens network the PBFT way", or something like that.